src/transformers/training_args_tf.py

```diff
@@ -111,7 +111,8 @@ class TFTrainingArguments(TrainingArguments):
         no_cuda (:obj:`bool`, `optional`, defaults to :obj:`False`):
             Whether to not use CUDA even when it is available or not.
         seed (:obj:`int`, `optional`, defaults to 42):
-            Random seed for initialization.
+            Random seed for initialization of the model's weights. To ensure reproducibility across runs, use the
+            :func:`~transformers.Trainer.model_init` function to instantiate the model.
```

Comment:
TFTrainer does not have the model_init argument.
```suggestion
            Random seed that will be set at the beginning of training.
```